go.mod

```diff
@@ -1,29 +1,29 @@
 module github.com/henvic/pgxtutorial
 
-go 1.22.4
+go 1.22
```

Comment:
This is important imo to not confuse language features and toolchain
(and not keep bumping this when nothing else changed, the build image/ci/... will)